ballsdex/packages/admin/info.py

```diff
@@ -16,6 +16,33 @@
 from ballsdex.settings import settings
 
 
+class PlayerInfoView(discord.ui.View):
+    def __init__(self, player: Player, username: str):
+        super().__init__()
+        self.player = player
+        self.username = username
+
+    @discord.ui.button(label="Recent Catches", style=discord.ButtonStyle.primary)
+    async def recently_caught(self, interaction: discord.Interaction, button: discord.ui.Button):
+        # Display the last 10 catches of the user, and how long it took for each catch
+        recent_balls = (
+            await BallInstance.filter(
+                player=self.player,
```

Comment:
Better filter the self-caught only. Also spawned time can be null (admin given), which results in an error, you also need to filter that out.
```suggestion
                player=self.player, spawned_time__isnull=False, trade_player=None
```